feature/gribi/ate_tests/backup_nhg_multiple_nh_test/backup_nhg_multiple_nh_test.go

```diff
@@ -324,20 +349,20 @@ func createFlow(t *testing.T, ate *ondatra.ATEDevice, top *ondatra.ATETopology,
 }
 
 // validateTrafficFlows verifies that the flow on ATE and check interface counters on DUT
-func validateTrafficFlows(t *testing.T, ate *ondatra.ATEDevice, flow *ondatra.Flow, drop bool, dPort []string) {
+func validateTrafficFlows(t *testing.T, ate *ondatra.ATEDevice, flow *ondatra.Flow, dPort []string) {
 	ate.Traffic().Start(t, flow)
 	time.Sleep(60 * time.Second)
 	ate.Traffic().Stop(t)
 	flowPath := gnmi.OC().Flow(flow.Name())
-	got := gnmi.Get(t, ate, flowPath.LossPct().State())
-	if drop {
-		if got != 100 {
-			t.Fatalf("Traffic passing for flow %s got %f, want 100 percent loss", flow.Name(), got)
-		}
-	} else {
-		if got > 0 {
-			t.Fatalf("LossPct for flow %s got %f, want 0", flow.Name(), got)
-		}
+	val, _ := gnmi.Watch(t, ate, flowPath.LossPct().State(), 5*time.Minute, func(val *ygnmi.Value[float32]) bool {
+		return val.IsPresent()
+	}).Await(t)
+	lossPct, present := val.Val()
+	if !present {
+		t.Fatalf("Could not read loss percentage for flow %q from ATE.", flow.Name())
+	}
+	if lossPct > 0.5 {
```

Comment:
Why 0.5% instead of 0%?

Reply:
Yes but we don't switch to backup with traffic in-flight. All modify operations are performed after the traffic is stopped.

Reply:
no, there is no modify operation here after the initial setup. here is about local repair. that's why I think we start with a relaxed bar (0.5%) is ok.

Reply:
add checks for interface status and remove the loss threshold to 0

Reply:
Thanks @mojiiba. Looks good!

Reply:
Actually, @LimeHat and @mojiiba you brought a good point. Reading the code again, it stops the traffic before the flipping the port status. 

What we wanted is that
1. Start traffic.
2. flip the port status.
3. Validate the traffic moved between ports. 

@mojiiba can your team help update the code? Thanks.